app/src/components/exploreFurther/exploreFurther.tsx

```diff
@@ -49,13 +50,20 @@ const ExploreFurther = () => {
             data-testid={`test-id-${index}`}
             mainActionLink={item.url}
             imageProps={{
-              alt: "",
-              aspectRatio: "sixteenByNine",
-              component: undefined,
+              component: (
+                <Image
+                  src={`/${item.image}`}
+                  alt={""}
+                  width={225}
+                  height={126}
+                  sizes="(max-width: 600px) 100vw, 25vw"
```

Comment:
Add `deviceSizes` in the config file to make use of this prop https://nextjs.org/docs/app/api-reference/components/image#devicesizes